numpy/ma/tests/test_core.py

```diff
@@ -2853,39 +2853,51 @@ def test_inplace_multiplication_array_type(self):
 
     def test_inplace_floor_division_scalar_type(self):
         # Test of inplace division
+        # Check for TypeError in case of unsupported types
+        unsupported = {np.sctypeDict[t] for t in np.typecodes["Complex"]}
```

Comment:
```suggestion
        unsupported = {np.dtype(t).type for t in np.typecodes["Complex"]}
```
Style suggestion; the same applies to `test_inplace_floor_division_array_type`.

Reply:
Thanks, I have updated in same commit.